crates/router/src/core/payouts/validator.rs

```diff
@@ -137,27 +148,27 @@ pub async fn validate_create_request(
         None
     };
 
-    // payout_token
-    let payout_method_data = match (req.payout_token.as_ref(), customer.as_ref()) {
-        (Some(_), None) => Err(report!(errors::ApiErrorResponse::MissingRequiredField {
-            field_name: "customer or customer_id when payout_token is provided"
-        })),
-        (Some(payout_token), Some(customer)) => {
-            helpers::make_payout_method_data(
-                state,
-                req.payout_method_data.as_ref(),
-                Some(payout_token),
-                &customer.customer_id,
-                merchant_account.get_id(),
-                req.payout_type,
-                merchant_key_store,
-                None,
-                merchant_account.storage_scheme,
-            )
-            .await
-        }
-        _ => Ok(None),
-    }?;
+    // // payout_token
```

Comment:
Why is this commented?

Reply:
match statement is updated and moved below.